refactor: :recycle: remove original extension and `.gz` from batch file names

signekb opened the pull request:
## Description

Since we won’t compress these files after all, now that we’ll store batch files as Parquet.

<!-- Select quick/in-depth as necessary -->
This PR needs a quick review.

## Checklist

- [X] Added or updated tests
- [X] Ran `just run-all`

signekb committed:
refactor: :recycle: remove `.gz` from batch file names

signekb committed:
Merge branch 'main' into refactor/remove-gz-from-batch-file-names

lwjohnst86 commented on src/seedcase_sprout/core/create_batch_file_name.py:
```diff
@@ -11,6 +11,6 @@ def create_batch_file_name(path: Path) -> str:
 
     Returns:
-        The created batch file name in the format {timestamp}-{uuid}{ext}.gz
+        The created batch file name in the format {timestamp}-{uuid}{ext}
```
Hm, I think we can drop the `{ext}` bit, I don't know what value that will provide. Thoughts?

martonvago replied:
Will this file name be passed to the Polars parquet writer function? Then it could have ".parquet" hardcoded in it? Or, yeah, drop the extension here and add it when writing to parquet.

signekb replied:
Ah, you’re right! If we’ll always store as parquet, then the original extension doesn’t have to be a part of the batch file name.

signekb replied:
@martonvago I like your second option above; add the extension when writing to parquet. Then, if we’ll change which format we’ll save to (for currently unknown reasons), the we don’t have to change it here.

martonvago commented on tests/core/test_create_batch_file_name.py:
Should we have parquet files in the tests (eventually), just to make it consistent with the design?

signekb committed:
Merge branch 'main' into refactor/remove-gz-from-batch-file-names

signekb committed:
refactor: remove original file extension from batch file names

Bc we don't need the extension anymore, we actually also don't need the path to the file itself. So, I have removed that as a function arg as well.

signekb committed:
test: :white_check_mark: change test batch file extensions to `.parquet`

martonvago reviewed (approved):
:tada:
